### PR TITLE
feat: Additional CSS changes for Virtual Cards

### PR DESCRIPTION
**Pull request by bistaastha**
### Description
Adds UI interaction for Copying of card number & cvv
### Walkthrough
copilot:walkthrough

## Clickup
https://app.clickup.com/t/86cuaht42

## Code Coverage
Please add code coverage here

## UI Preview
Please add screenshots for UI changes

### CODE REVIEW

File: src/app/shared/components/virtual-card/virtual-card.component.html [hunk 1]
```diff
@@ -18,8 +18,12 @@
       <div>******</div>
     </div>
     <div *ngIf="showCardNumber" class="virtual-card__card-number__card-number-unmasked">
-      <div>cardNumber.slice(0, 4)</div>
-      <div>cardNumber.slice(4, 10)</div>
+      <div>
```
Comment (bistaastha) [inline src/app/shared/components/virtual-card/virtual-card.component.html:21]: Same preview as this: https://github.com/fylein/fyle-mobile-app/pull/2698/files#r1464325054

Comment (bistaastha) [reply src/app/shared/components/virtual-card/virtual-card.component.html:21]: Video Preview:
https://github.com/fylein/fyle-mobile-app/assets/40273492/9a126e3f-2518-4f54-9802-2e07bb999bd4

**Comment by mvaishnavi** [comment]
my review not needed; please merge in once approved by the mobile app team.

File: src/app/shared/components/virtual-card/virtual-card.component.html (hunk 1 above)
Comment (Julias0) [reply src/app/shared/components/virtual-card/virtual-card.component.html:21]: slice pipe. Refer to other PR

File: src/app/shared/components/virtual-card/virtual-card.component.scss [hunk 2]
```diff
@@ -57,11 +58,27 @@
       font-size: 14px;
       line-height: 16px;
       color: $blue-black;
+      letter-spacing: 0.022px;
```
Comment (Julias0) [inline src/app/shared/components/virtual-card/virtual-card.component.scss:61]: Why is this needed?

**Review by Julias0 — CHANGES_REQUESTED** [review]
comments

File: src/app/shared/components/virtual-card/virtual-card.component.scss (hunk 2 above)
Comment (bistaastha) [reply src/app/shared/components/virtual-card/virtual-card.component.scss:67]: @Julias0 We can discuss this further if needed

File: src/app/shared/components/virtual-card/virtual-card.component.scss [hunk 3]
```diff
@@ -65,15 +70,30 @@
...
       line-height: 16px;
+      width: 20px;
+      height: 16px;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+    }
+    &__cvv-number {
+      font-size: 12px;
+      line-height: 1.3;
```
Comment (Julias0) [inline src/app/shared/components/virtual-card/virtual-card.component.scss:86]: avoid line height

**Review by Julias0 — APPROVED** [review]

**Pull request merged**